egs/mobvoi/v1/local/split_datasets.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+stage=0
+
+. ./cmd.sh
+. ./path.sh
+. utils/parse_options.sh
+
+set -eu
+
+
+if [ $stage -le 1 ]; then
+  dir=data/download/ticmini2_dataset_20180607
+  trans=$dir/hixiaowen.txt
+  paste -d ' ' <(cat $trans | awk '{print $1}' | awk '{split($1,a,"."); print a[1]}') <(cat $trans | cut -d ' ' -f2-) > $dir/hixiaowen_text || exit 1
```

Comment:
(cat $trans ~~| awk '{print $1}'~~ | awk '{split($1,a,"."); print a[1]}')

`| awk '{print $1}'` is not needed.